dealership.rb

```diff
@@ -1,29 +1,58 @@
+require 'csv'
+
+module CarLoader
+  def self.get_cars_from_csv(filepath)
+    @cars = CSV.read(filepath, { :headers => true, :header_converters => :symbol}).map { |car_object| Car.new(car_object) }
```

Comment:
You don't need to set this to a variable at all here. All you are doing is returning the array of car objects.